lib/process-services/src/lib/process-list/services/process-filter.service.ts

```diff
@@ -116,12 +118,14 @@ export class ProcessFilterService {
                 (res) => {
                     const filters: FilterProcessRepresentationModel[] = [];
                     res.forEach((filter) => {
-                        if (filter.name === runningFilter.name) {
-                            filters.push(new FilterProcessRepresentationModel({ ...filter, filter: runningFilter.filter, appId }));
-                        } else if (filter.name === completedFilter.name) {
-                            filters.push(new FilterProcessRepresentationModel({ ...filter, filter: completedFilter.filter, appId }));
-                        } else if (filter.name === allFilter.name) {
-                            filters.push(new FilterProcessRepresentationModel({ ...filter, filter: allFilter.filter, appId }));
+                        if (!this.isFilterAlreadyExisting(filters, filter.name)) {
```

Comment:
This new condition should be tested as part of `createDefaultFilters` method as well, to ensure this bug won't happen again